deploy/crds/aws_v1alpha1_awsfederatedaccountaccess_cr.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: aws-account-operator
 spec:
   awsCustomerCredentialSecret: 
-    name: osd-creds-mgmt-w2tclb-secret
-    namespace: aws-account-operator
-  externalCustomerAWSIAMARN: arn:aws:iam::414830704956:user/james.harrington
+    name: secret-name
+    namespace: example-namespace
+  externalCustomerAWSIAMARN: arn:aws:iam::{$AWS_ACCOUNT_ID}:user/{$AWS_IAM_USER}
```

Comment:
Are these meant to be templatable? either way `${AWS_ACCOUNT_ID}` seems better?

Reply:
Not intended specifically to be templateable. Just to indicate where to place what